__test__/timeTravel.test.js

```diff
@@ -0,0 +1,31 @@
+
+import TimeTravel from '../src/app/components/timeTravel'
+import React from 'react'
+import { configure, shallow, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+configure({ adapter: new Adapter() });
+
+describe('Time travel rendered correctly', () => {
+    let wrapper
+    let props = {
+        currentState:[{}],
+        handelPlay : () => 'hello world',
+        logofTime: [['App',1.5]],
+        index:0
+    }
+    beforeAll(() => {
+        wrapper = shallow(<TimeTravel 
+            currentState = { props.currentState }
+            handelPlay = { props.handelPlay }
+            logofTime = { props.logofTime }
+            index = { props.index }
+            />)
+    })
+    xit('Timetravel rendered correctly', () => {
```

Comment:
is this test not working?

Reply:
Yes, they work correctly I commented them because while I was writing other tests so that I can have a cleaner terminal log. 
I will uncomment them and make an other pull request.